Some graphs are not rendering correctly after the graphql API change:

```
Fetching graph
Traceback (most recent call last):
  File "./zcash-issue-dag.py", line 269, in <module>
    main()
  File "./zcash-issue-dag.py", line 62, in main
    dg = nx.compose_all([
  File "/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/networkx/algorithms/operators/all.py", line 179, in compose_all
    raise ValueError("cannot apply compose_all to an empty list")
ValueError: cannot apply compose_all to an empty list
Error: Process completed with exit code 1.
```

--- a/helpers/zenhub.py
+++ b/helpers/zenhub.py
@@ -2,12 +2,12 @@ import networkx as nx
 from sgqlc.endpoint.http import HTTPEndpoint
 from sgqlc.operation import Operation
 
-from helpers.github import CORE_REPOS, WALLET_REPOS, ZF_REPOS, ZF_FROST_REPOS
+from helpers.github import CORE_REPOS, TFL_REPOS, WALLET_REPOS, ZF_REPOS, ZF_FROST_REPOS
 from zenhub_schema import zenhub_schema
 
 WORKSPACE_SETS = {
     # ecc-core
-    '5dc1fd615862290001229f21': CORE_REPOS.keys(),
+    '5dc1fd615862290001229f21': list(CORE_REPOS.keys()) + list(TFL_REPOS.keys()),
     # ecc-wallet
     '5db8aa0244512d0001e0968e': WALLET_REPOS.keys(),
     # zf

--- a/zcash-issue-dag.py
+++ b/zcash-issue-dag.py
@@ -21,8 +21,12 @@ DAG_VIEW = os.environ.get('DAG_VIEW', 'core')
 REPOS = github.REPO_SETS[DAG_VIEW]
 
 WORKSPACES = {
-    workspace_id: [repo_id for repo_id in repos if repo_id in REPOS]
-    for (workspace_id, repos) in zenhub.WORKSPACE_SETS.items()
+    workspace_id: repos
+    for (workspace_id, repos) in {
+        workspace_id: [repo_id for repo_id in repos if repo_id in REPOS]
+        for (workspace_id, repos) in zenhub.WORKSPACE_SETS.items()
+    }.items()
+    if len(repos) > 0
 }
 
 SUPPORTED_CATEGORIES = set(['releases', 'targets'])
@@ -57,12 +61,15 @@ def main():
     gapi = github.api(GITHUB_TOKEN)
     zapi = zenhub.api(ZENHUB_TOKEN)
 
+    if len(WORKSPACES) == 0:
+        print('Error: DAG_VIEW="{}" has no matching ZenHub workspaces'.format(DAG_VIEW))
+        return
+
     # Build the full dependency graph from ZenHub's per-workspace API.
     print('Fetching graph')
     dg = nx.compose_all([
         zenhub.get_dependency_graph(zapi, workspace_id, repos)
         for (workspace_id, repos) in WORKSPACES.items()
-        if len(repos) > 0
     ])
 
     print('Rendering DAG')
@@ -70,8 +77,7 @@ def main():
     if SHOW_EPICS:
         epics_issues = []
         for (workspace_id, repos) in WORKSPACES.items():
-            if len(repos) > 0:
-                epics_issues += zenhub.get_epics(zapi, workspace_id, repos)
+            epics_issues += zenhub.get_epics(zapi, workspace_id, repos)
         epics_issues = set(epics_issues)
 
         epics_mapping = github.download_issues(gapi, [gh_ref for (_, gh_ref) in epics_issues], REPOS)

